### PR TITLE
Development fixing k8s tests

### PR DESCRIPTION
**Pull request by khaledyoussef24**
### Description

fixes for k8s tests
### Changes

k8s test
k8s with module
nightly timeout

### Related Issues
#658

### CODE REVIEW

File: integrationtests/k8s_test.go [hunk 1]
```diff
@@ -200,7 +211,23 @@ func TestK8s(t *testing.T) {
...
+
+		// Check that master and workers is reachable
+		// testing connection on port 22, waits at max 3mins until it becomes ready otherwise it fails
+		ok = TestConnection(masterIP, "22")
+		assert.True(t, ok)
+
+		ticker := time.NewTicker(2 * time.Second)
+		for now := time.Now(); time.Since(now) < 1*time.Minute; {
+			<-ticker.C
+			output, err := RemoteRun("root", masterIP, "zinit list", privateKey)
```
Comment (MarioBassem) [inline integrationtests/k8s_test.go:226]: this checks if the k3s service is running on the master node, which i believe does not say anything about the other nodes in the cluster. imo it should not be the test's purpose to check if the flist works, but i also think that this it's just easier to check if the cluster works or not.

Comment (ashraffouda) [reply integrationtests/k8s_test.go:226]: yes @khaledyoussef24 plz use kubectl get nodes instead and make sure all workers are running

**Review by ashraffouda — CHANGES_REQUESTED** [review]

**Comment by xmonader** [comment]
This didn't move for 3 month? 

Please separate the k8s module fixes from anything else and skip it from tests

**Comment by khaledyoussef24** [comment]
There are seprate issue for k8s with module now and k8s test works fine we may close this issue

**Comment by xmonader** [comment]
Alright, please close the irrelevant ones

**Pull request closed without merging**